client/src/components/questionsAnswers/Search.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// import img from '../../assets/images/searchIcon.png';
```

Comment:
Is there a reason line 2 is commented out? Please consider omitting or if there is a significant purpose of leaving that comment in, please clarify within the comment!